crates/std/veryl/src/synchronizer/synchronizer_basic.veryl

```diff
@@ -0,0 +1,93 @@
+/// Basic synchronizer implementation using multi-bit FF scheme
```

Comment:
I think "multi-bit FF" generally means multi-bit FF cell.

https://www.techsimplifiedtv.in/2022/12/what-is-multi-bit-flip-flop-mbff-in-vlsi.html

So "Basic synchronizer implementation using single-bit FF scheme" is correct because this implementation doesn't use MBFF cell.

Reply:
Thanks for your review.
I modified the comment.